src/components/button/_mixins.scss

```diff
@@ -21,7 +21,7 @@
   border-radius: $button-border-radius;
   text-align: left;
   text-decoration: none;
-  transition: $duration--fast-01 motion(entrance, productive);
+  transition: all $duration--fast-02 motion(standard, productive);
```

Comment:
For buttons, I actually made this change during the last couple of weeks of release. My bad it wasn't reflected in the boxnote! The difference is really neglible but the original will be more compliant with the theory. Should we just update to `all $duration--fast-01 motion(entrance, productive)`?

Reply:
Done 👍